bin/reth/src/args/utils.rs

```diff
@@ -17,7 +17,7 @@ use reth_primitives::{DEV, GOERLI, HOLESKY, MAINNET, SEPOLIA};
 
 #[cfg(feature = "optimism")]
 /// Chains supported by op-reth. First value should be used as the default.
-pub const SUPPORTED_CHAINS: &[&str] = &["base", "base_goerli", "base_sepolia"];
+pub const SUPPORTED_CHAINS: &[&str] = &["base", "base-goerli", "base-sepolia"];
```

Comment:
this is a breaking change in the CLI right, we should communicate that in the release notes

Reply:
yeah but this now mirrors what op-geth accepts so this is good change